### ISSUE TITLE
NoMoreBugCheck - Tampering Whith Audio Drivers?

### ISSUE THREAD

**Issue by BlacknetSystem** [NSG650/NoMoreBugCheck#2]
Hello Im A User who has a steam deck and windows is installed on it ( 11 ) does when a audio drivers crash causes a bsod im curious i havent tested it yet my drivers ( amd audio drivers ) high def

**Comment by NSG650** [comment]
Audio drivers usually run in userspace now a days in Windows so an audio driver crash shouldn't really cause a BSOD. I wouldn't recommend you running this driver as well since if an actual crash occurs in a device driver it might damage the hardware. So please refrain from running this on real hardware. Also Windows 11 on a steam deck? (you're weird (jk))

**Comment by BlacknetSystem** [comment]
also a problem here?
![image](https://github.com/NSG650/NoMoreBugCheck/assets/154022385/be9bcf1e-61bf-4b62-a053-c6eec428e257)

**Comment by BlacknetSystem** [comment]
I moved the file here i tested whith space and whitout the " but still keep on the bug here

**Comment by NSG650** [comment]
You have to use `sc.exe`.` sc` in powershell is entire different. 
If you're in powershell use `sc.exe` or in cmd simply `sc` or `sc.exe`

**Comment by LagLifeYT** [comment]
> I wouldn't recommend you running this driver as well since if an actual crash occurs in a device driver it might damage the hardware. So please refrain from running this on real hardware. 

I may have been responsible for a few people running this on real hardware, since a couple months ago I posted [this](https://youtu.be/nWmdia-EBiA) video which unexpectedly got 2.6K views and probably drove a few dozen people here. 

I did at least add a warning to not run this anywhere since it will almost certainly cause data loss if Windows "crashes" but didn't add that it could cause hardware damage since I didn't think anyone would be running this outside a VM. 
Maybe a much more visible warning should be added to the README so nobody tries to seriously use this

**Comment by NSG650** [comment]
> > I wouldn't recommend you running this driver as well since if an actual crash occurs in a device driver it might damage the hardware. So please refrain from running this on real hardware. 
> 
> I may have been responsible for a few people running this on real hardware, since a couple months ago I posted [this](https://youtu.be/nWmdia-EBiA) video which unexpectedly got 2.6K views and probably drove a few dozen people here. 
> 
> I did at least add a warning to not run this anywhere since it will almost certainly cause data loss if Windows "crashes" but didn't add that it could cause hardware damage since I didn't think anyone would be running this outside a VM. 
> Maybe a much more visible warning should be added to the README so nobody tries to seriously use this

True, I should add such a warning. Thank you got checking my projects out as well! It means alot to me!!!

**Comment by ibay770** [comment]
What kind of damage can happen? Never heard of a possibility. Running it and it works fine.
